therr-services/maps-service/src/handlers/spaceMetrics.ts

```diff
@@ -110,10 +118,15 @@ const getSpaceMetrics = (req, res) => {
                 startDate,
                 endDate,
             } = req.query;
-
-            return Store.spaceMetrics.getForDateRange(startDate, endDate, {
-                spaceId,
-            }).then((results) => res.status(200).send({ space, metrics: results }));
+            return getMetrics(startDate, endDate, spaceId).then((results) => res.status(200).send({
```

Comment:
Can you use [destructuring assignmen](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)t here?

`.then(([currentMetrics, previousMetrics]) => ... `